app/src/main/java/com/brainwallet/tools/sqlite/CurrencyDataSource.java

```diff
@@ -31,6 +32,27 @@ public class CurrencyDataSource implements BRDataSourceInterface {
             BRSQLiteHelper.CURRENCY_RATE
     };
 
+    /// Set the most popular fiats
```

Comment:
The view I have doesnt have the picker @andhikayuana .  Will you add?

Reply:
![Screenshot 2025-05-07 at 08 29 21](https://github.com/user-attachments/assets/78ded2c1-efc3-438e-a6e4-dbe03f20f6bc)

Reply:
actually pls just pull on this branch